sh_matching_analysis/scripts/parse_matches_html.py

```diff
@@ -76,30 +79,33 @@
             sh_taxonomy = row[4]
             compound_code = row[5]
 
-            # count seqs
-            if sh_code in seqs_status_hash:
-                seqs_status_hash[sh_code] += 1
-            else:
-                seqs_status_hash[sh_code] = 1
+            # # count seqs
+            # if sh_code in seqs_status_hash:
+            #     seqs_status_hash[sh_code] += 1
+            # else:
+            #     seqs_status_hash[sh_code] = 1
```

Comment:
Let's not keep the commented out blocks of code unless it's for temporary testing purposes.